### PR TITLE
Hypha

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

File: packages/element/src/itk-remote-viewport.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    this.remote = remote;
+    this.frame = new SelectorController(
+      this,
+      this.remote,
+      (state) => state?.context.frame
+    );
+    const p = new Promise((resolve) => setTimeout(resolve, 5000));
+    p.then(() => {
+      this.remote.send({ type: 'render' });
+    });
```
Comment (PaulHax) [inline packages/element/src/itk-remote-viewport.ts:39]: remove wait

File: packages/remote-viewport/src/hyphaWebsocketClient.d.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+declare namespace hyphaWebsocketClient {
```
Comment (PaulHax) [inline packages/remote-viewport/src/hyphaWebsocketClient.d.ts:1]: use npm package

File: packages/viewer/src/viewport-machine.ts [hunk 3]
```diff
@@ -54,7 +58,7 @@ export const viewportMachine = createMachine({
                 context.cameraSubscription.unsubscribe();
               context.cameraSubscription = context.camera?.subscribe(
                 (state) => {
-                  self.send({
+                  (self as ActorRef<SetCameraEvent | SetCameraPoseEvent>).send({
```
Comment (thewtex) [inline packages/viewer/src/viewport-machine.ts:61]: Remote `SetCameraPoseEvent`?

**Review by thewtex — APPROVED** [review]

File: packages/viewer/src/viewport-machine.ts (hunk 3 above)
Comment (PaulHax) [reply packages/viewer/src/viewport-machine.ts]: I remember now what `SetCameraPoseEvent` does and added this comment:

Let observers of Viewport know that camera has updated

**Pull request merged**